Pratiksha/Bootstrap/Assignment5/Assignment5.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+
+</head>
+<body>
+    <div class="container">
+        <nav aria-label="breadcrumb">
+            <ol class="breadcrumb m-3">
+              <li class="breadcrumb-item"><a href="#">Home</a></li>
+              <li class="breadcrumb-item"><a href="#">Employee</a></li>
+              <li class="breadcrumb-item active" aria-current="page">Employee Details
+              </li>
+            </ol>
+          </nav>
+          <h1>List of Employee<span class="badge badge-secondary">New</span></h1>
+          <table class="table">
+            <thead class="thead-light">
+              <tr>
```

Comment:
here 2 spaces from parent tag not consistent please check 
required 4 spaces from parent tag to child tag 
check this in each and every file